src/main/java/org/jenkinsci/plugins/saml/SamlFileResource.java

```diff
@@ -61,11 +65,36 @@ public boolean exists() {
         return getFile().exists();
     }
 
+    @Override
+    public boolean isReadable() {
+        return getFile().canRead();
+    }
+
+    @Override
+    public boolean isOpen() {
+        return false;
+    }
+
+    @Override
+    public URL getURL() {
+        throw new NotImplementedException();
+    }
+
+    @Override
+    public URI getURI() {
+        throw new NotImplementedException();
+    }
+
     @Override
     public String getFilename() {
         return fileName;
     }
 
+    @Override
+    public String getDescription() {
+        return null;
```

Comment:
maybe return the name of the file instead to return a null

Reply:
Thank you, will change it